scripts/talos/README.md

```diff
@@ -0,0 +1,53 @@
+# Talos
+
+## Description
+
+Talos is a bumper bot that runs automatically for Psammead packages.
```

Comment:
Can you please explain here why Talos is needed? If you need more context, Drew's talk from last week can be a useful source.